.github/workflows/ci.yaml

```diff
@@ -185,7 +144,6 @@ jobs:
                     --bazelrc=${GITHUB_WORKSPACE//\\/\/}/.aspect/bazelrc/ci.bazelrc \
                     --bazelrc=${GITHUB_WORKSPACE//\\/\/}/.github/workflows/ci.bazelrc \
                     test \
-                    --config=${{ matrix.config }} \
```

Comment:
Need to also change
```
common:local --disk_cache=~/.cache/bazel-disk-cache
```
to
```
common --disk_cache=~/.cache/bazel-disk-cache
```
in `.github/workflows/ci.bazelrc` since you're no longer setting the `--config=local` flag